make/common/templates/clair/config.yaml

```diff
@@ -2,7 +2,7 @@ clair:
   database:
     type: pgsql
     options:
-      source: postgresql://$username:$password@$host:$port?sslmode=disable
+      source: postgresql://$username:$password@$host:$port/$dbname?sslmode=disable
```

Comment:
Does this change impact on the upgrade script?
and is the default dbname in harbor.cfg is set in another patch?

Reply:
The default name is already there before this patch, We just not use it.